### PR TITLE
Fix S3 Setup

### PR DESCRIPTION
**Pull request by ghickman**
This better organises the S3 bucket by using a bucket related to the series we're currently working on (`v0.5`), uses an IAM with permissions tailored only to that bucket, and splits backups and data into `backups` and `csvs` "directories" in the bucket.

The backup filenames have been changed to avoid this situation:

### CODE REVIEW

File: scripts/backup.py [hunk 1]
```diff
@@ -17,22 +17,22 @@ def dump_database(db_name, db_user, backup_name):
...
     """
     s3 = boto3.resource("s3")
-    s3.Bucket(bucket_name).upload_file(full_file_name, file_name)
+    s3.Bucket(bucket_name).upload_file(local_path, f"backups/{key}")
 
 
 def main(db_name, db_user, backups_dir, bucket_name):
-    today = datetime.now().strftime("%d-%m-%Y")
-    backup_name = f"{db_name}-{today}.dump"
+    today = datetime.now().strftime("%Y-%m-%Y")
```
Comment (fredkingham) [inline scripts/backup.py]: Year month year?

Comment (ghickman) [reply scripts/backup.py]: 🤦‍♂️

Comment (ghickman) [reply scripts/backup.py]: Fixed and rebased

**Comment by fredkingham** [comment]
👍

**Pull request merged**